gobblin-service/src/test/java/org/apache/gobblin/service/modules/scheduler/GobblinServiceJobSchedulerTest.java

```diff
@@ -338,6 +338,7 @@ public void testJobSchedulerAddFlowQuotaExceeded() throws Exception {
     serviceLauncher.addService(flowCatalog);
     serviceLauncher.start();
 
+    // We need to test adhoc flows since scheduled flows do not have a quota check in the scheduler
```

Comment:
I might be missing something here but even scheduled flows with RUN IMMEDIATELY set to true still follows this behavior?

Reply:
When I tried to test scheduled run immediately flows I saw this code block and see its only for adhoc https://github.com/apache/gobblin/blob/99a1aae8940d1854d4de83d726013e4972d71c55/gobblin-service/src/main/java/org/apache/gobblin/service/modules/scheduler/GobblinServiceJobScheduler.java#L565